api/openapi-spec/v1.0.yaml

```
@@ -826,6 +828,96 @@ paths:
         default:
           $ref: '#/components/responses/error'
       x-ms-docs-operation-type: operation
+  '/users/{user-id}/appRoleAssignments':
+    description: Provides operations to assign global roles to users.
+    get:
+      tags:
+        - user.appRoleAssignment
+      summary: Get appRoleAssignments from a user
+      description: Represents the global roles a user has been granted for an application.
+      operationId: user.ListAppRoleAssignments
+      responses:
+        200:
+          description: Retrieved appRoleAssignments
+          content:
+            application/json:
+              schema:
+                title: Collection of appRoleAssignments
+                type: object
+                properties:
+                  value:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/appRoleAssignment'
+                    maxItems: 100
+                  '@odata.nextLink':
+                    type: string
+        default:
+          $ref: '#/components/responses/error'
+    post:
+      tags:
+        - user.appRoleAssignment
+      summary: Grant an appRoleAssignment to a user
+      description: 'Use this API to assign a global role to a user. To grant an app role assignment to a user, you need three identifiers:'
```

Nit: I think the `To grant an app role assignment to a user, you need three identifiers:` doesn't belong here. I guess this will look wrong in the swagger UI. (If it doesn't, that might just be pure luck ;) )

---

It seems this came from https://learn.microsoft.com/en-us/graph/api/user-post-approleassignments?view=graph-rest-1.0&tabs=http
```suggestion
      description: |
        Use this API to assign a global role to a user. To grant an app role assignment to a user, you need three identifiers:
        * `principalId`: The `id` of the user to whom you are assigning the app role.
        * `resourceId`: The `id` of the resource `servicePrincipal` or `application` that has defined the app role.
        * `appRoleId`: The `id` of the `appRole` (defined on the resource service principal or application) to assign to the user.
```